src/react/components/StyleEditor/constants.js

```diff
@@ -39,3 +39,38 @@ export const LINE_STYLES = {
         }
     ]
 };
+
+
+// Default values for every style settings value
+export const PREVIEW_DEFAULTS = {
```

Comment:
How does preview defaults differ from the default style that is used as template for StyleEditor? Do we really need these or could we use that default style here as well?

Reply:
Removed `PREVIEW_DEFAULTS` and replaced with `OSKARI_BLANK_STYLE`